### PR TITLE
Fix rubocop offenses

### PR DESCRIPTION
**Pull request by pschambacher**
## Description

Not sure why but Rubocop suddenly started breaking. This fixes most of the offenses as they are nitpicks.
I extended the maximum line length mostly because of https://github.com/zendesk/kasket/blob/master/lib/kasket/query_parser.rb#L15

Fixing rubocop was enough yak shaving for me... refactoring how we regenerate this regex will be a dragon for another person.

### CODE REVIEW

**Review by mariozaizar — APPROVED** [review]

File: gemfiles/common.rb [hunk 1]
```diff
@@ -3,6 +3,6 @@
 
 gem 'activerecord-jdbcmysql-adapter', '~> 1.2', platforms: :jruby
 gem 'byebug', platforms: :ruby
-gem 'rubocop', '~> 0.49.1', platforms: :ruby
+gem 'rubocop', '~> 0.89.1', platforms: :ruby
```
Comment (mariozaizar) [inline gemfiles/common.rb:6]: ❤️

**Review by codealchemy — APPROVED** [review]
👍

**Review by bquorning — APPROVED** [review]

File: lib/kasket.rb [hunk 2]
```diff
@@ -33,19 +33,24 @@ def setup(options = {})
...
+      Arel::SelectManager.include Kasket::SelectManagerMixin
     end
   end
 
   def self.cache_store=(options)
     @cache_store = ActiveSupport::Cache.lookup_store(options)
   end
 
-  def self.cache
+  def self.cache_store
```
Comment (bquorning) [inline lib/kasket.rb:45]: Why the rename of this method?

**Review by bgatelet — APPROVED** [review]

**Review by angelim — APPROVED** [review]

File: lib/kasket.rb (hunk 2 above)
Comment (pschambacher) [reply lib/kasket.rb:45]: Rubocop isn't happy that the memoize variable and the method name aren't the same...

**Pull request merged**